docs/visualize/goal.asciidoc

```diff
@@ -0,0 +1,35 @@
+[[goal-chart]]
+== Goal and Gauge
+
+A goal visualization displays how your metric progresses toward a fixed goal. Gauge visualization displays in which 
+predefined range falls your metric.
+
+include::y-axis-aggs.asciidoc[]
+
+You can click the *Advanced* link to display more customization options:
+
+*JSON Input*:: A text field where you can add specific JSON-formatted properties to merge with the aggregation
+definition, as in the following example:
+
+[source,shell]
+{ "script" : "doc['grade'].value * 1.2" }
+
+NOTE: In Elasticsearch releases 1.4.3 and later, this functionality requires you to enable
+{ref}/modules-scripting.html[dynamic Groovy scripting].
+
+The availability of these options varies depending on the aggregation you choose.
+
+Click the *Options* tab to change the following options:
+
+- *Gauge Type* you can select between arc, circle and metric display type.
+- *Percentage Mode* will show all values as percentages
+- *Vertical Split* will put the gauges one under another instead of one next to another
+- *Show Labels* selects if you want to show the labels or not
+- *Sub Text* text for the label that appears below the value
+- *Auto Extend Range* automatically grows the gauge if value is over its extents.
+- *Ranges* you can add custom ranges. Each range will get assigned a color. If value falls within that range it will get
+assigned that color. Chart with single range is called goal chart. Chart with multiple ranges is called gauge chart.
+- *Color Options* define how to color your ranges (which color schema to use). Color options are only visible if more than
+one range is defined.
+- *Style - Show Scale* shows or hides the scale
+- *Style - Color Labels* should the labels get the color of the current values range
```

Comment:
`whether the labels should have the same color as the range where the value falls in`